src/pages/upgrades/get-involved/index.tsx

```diff
@@ -15,153 +33,207 @@ import PageMetadata from "../../../components/PageMetadata"
+      {...props}
+    />
+  )
+}
 
-const StyledBreadcrumbs = styled(Breadcrumbs)`
-  justify-content: center;
-`
+const SloganGradient = (props: ChildOnlyProp) => {
+  const upgradesGradient = useToken("colors", "upgradesGradient")
 
-const StyledGrayContainer = styled(GrayContainer)`
-  margin-bottom: 3rem;
-  padding-bottom: 2rem;
-  border-bottom: 1px solid ${(props) => props.theme.colors.border};
-`
+  return (
+    <Box
+      fontSize={{ base: "2.5rem", lg: "5xl" }}
+      fontWeight="extrabold"
+      lineHeight="140%"
+      maxW="720px"
+      mt={4}
+      mb={0}
+      mx="auto"
+      bgImage={upgradesGradient}
+      bgClip="text"
+      sx={{
+        "&": {
+          "-webkit-text-fill-color": "transparent",
+        },
+      }}
+      {...props}
+    />
+  )
+}
 
-const StyledCalloutBanner = styled(CalloutBanner)`
-  background: transparent;
-  @media (max-width: ${(props) => props.theme.breakpoints.m}) {
-    width: 100%;
-    padding: 0rem;
-    padding-top: 4rem;
-    margin-left: 0rem;
-  }
-`
+const Subtitle = (props: ChildOnlyProp) => (
+  <Flex
+    justify="center"
+    textAlign="center"
+    color="text200"
+    lineHeight="140%"
+    fontSize="2xl"
+    mt={4}
+    {...props}
+  />
+)
+
+const HeroContainer = (props: ChildOnlyProp) => (
+  <Box
+    pl={0}
+    pt={{ base: 8, lg: 32 }}
+    pb={32}
+    w={{ base: "full", lg: "50%" }}
+    {...props}
+  />
+)
+
+const HeroCard = (props: ChildOnlyProp) => (
+  <Flex
+    justify="center"
+    textAlign="center"
+    mt={{ base: 4, lg: -8 }}
+    {...props}
+  />
+)
+
+const StyledCard = (props: ComponentPropsWithRef<typeof Card>) => (
+  <Card flex="1 1 30%" minW="240px" m={4} p={6} {...props} />
+)
+
+const StyledCardList = (props: ComponentPropsWithRef<typeof CardList>) => (
+  <Box as={CardList} mr={8} {...props} />
+)
+
+const LeaderboardContainer = (props: ChildOnlyProp) => (
+  <Flex
+    align="center"
+    direction="column"
+    w={{ base: "full", md: "50%" }}
+    pl={0}
+    pt={8}
+    {...props}
+  />
+)
+
+const Row = (props: ChildOnlyProp) => (
+  <Flex align="center" direction={{ base: "column", md: "row" }} {...props} />
+)
+
+const ReverseRow = (props: ChildOnlyProp) => (
+  <Flex
+    align="center"
+    direction={{ base: "column-reverse", md: "row" }}
+    {...props}
+  />
+)
+
+const Column = (props: ChildOnlyProp) => (
+  <Box
+    w={{ base: "full", md: "50%" }}
+    ml={16}
+    mr={{ base: 16, md: 0 }}
+    {...props}
+  />
+)
+
+const Staking = (props: ChildOnlyProp) => (
+  <Flex
+    direction="column"
+    w="full"
+    mt={8}
+    mb={-8}
+    p={16}
+    background="cardGradient"
+    {...props}
+  />
+)
+
+const StyledCalloutBanner = (
+  props: ComponentPropsWithRef<typeof CalloutBanner>
+) => (
+  <Box
+    as={CalloutBanner}
+    bg="transparent"
+    w={{ base: "full", md: "auto" }}
+    p={{ base: 0, md: 12 }}
+    pt={{ base: 16, md: 12 }}
+    sx={{
+      "&": {
+        ml: 0,
+      },
```

Comment:
At the moment, `CalloutBanner` is written in emotion and the *margins* of the component can't be overridable using `ml=0` because `@media (max-width)` query gets higher precedence over the Chakra class. But Chakra `sx` attribute allows us to override the `@media (max-width)` query, that's why I used it here.